### PR TITLE
COBRA-1820

### PR DESCRIPTION
**Pull request by sbeck14**
Added command - 'addons:plans:info'
Fixed formatting for addons:info on new attached_to property of addon info object

### CODE REVIEW

File: lib/terminal.js [hunk 1]
```diff
@@ -179,7 +179,12 @@ function vtable(obj, sort) {
...
     if(obj[d] && typeof(obj[d]) === 'object') {
       var subkeys = sort ? Object.keys(obj[d]).sort() : Object.keys(obj[d]);
-      obj[d] = subkeys.map(function(z) { return z + '=' + (obj[d][z] ? obj[d][z].toString() : '') }).join(', ');
+      obj[d] = subkeys.map(function(z) { 
+        if (obj[d][z] && typeof(obj[d][z]) !== 'string'){
+          return Object.keys(obj[d][z]).map(y => (y + '=' + obj[d][z][y].toString())).join(', ');
+        }
+        return z + '=' + (obj[d][z] ? obj[d][z].toString() : '') 
+      }).join(', ');
     }
```
Comment (trevorlinton) [inline lib/terminal.js]: Nice.

File: plugins/addons/index.js [hunk 2]
```diff
@@ -68,6 +68,30 @@ function list_addons_plans(appkit, args) {
...
+
+function format_plan_info(addon_service) {
+  let apps = [];
+  addon_service.provisioned_by.map((app) => ( apps.push(`   • Name: ${app.name}\n     ID: ${app.id}`) ));
+  return `**+ ${addon_service.human_name} (${addon_service.name}) \$${addon_service.price.cents/100}/${addon_service.price.unit}**
+  ***Id:*** ${addon_service.id}
+  ***State:*** ${addon_service.state}
+  ***Description:*** ${addon_service.description}
+  ***Attached Apps (${apps.length}):*** \n${ apps.length != 0 ? `${apps.join('\n\n')}\n` : '' }`;
+}
```
Comment (trevorlinton) [inline plugins/addons/index.js]: Can we change this label to `Provisioned By:`  from `Attached Apps` since this is the plan and not the addon?

**Comment by trevorlinton** [comment]
Did we add into this the functionality to see which apps are attached to which addons in the `addons:info` command? I'm not seeing it.

**Comment by sbeck14** [comment]
screen shot 2018-07-19 at 9 02 33 am

Our change to the controller API adds an "attached_to" property that is displayed in addons:info, is that what you wanted?

**Pull request merged**